src/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+
+#include "Server.h"
+
+constexpr int EXPECTED_ARGUMENTS = 2;
+
+int main(const int argc, char **argv)
+{
+    if (EXPECTED_ARGUMENTS != argc)
+    {
+        std::cout << "Usage: " << argv[0] << " [port]" << std::endl;
+        return EXIT_FAILURE;
+    }
+    const int server_port = std::stoi(argv[1]);
```

Comment:
create a wrapper that will give u the args as u need rather then explicitly pulling it from the list with magics